package.json

```diff
@@ -27,22 +26,21 @@
     "uuid": "^3.3.3"
   },
   "devDependencies": {
-    "@types/chai": "^4.2.6",
-    "@types/command-line-args": "^5.0.0",
+    "@types/chai": "^4.2.7",
     "@types/mocha": "^5.2.7",
     "@types/sinon": "^7.5.1",
     "@types/uuid": "^3.4.6",
+    "@types/yargs": "^13.0.3",
     "chai": "^4.2.0",
-    "chai-as-promised": "^7.1.1",
     "chalk": "^2.4.2",
-    "command-line-args": "^5.1.1",
     "globby": "^10.0.1",
     "mocha": "^6.2.2",
-    "simple-git": "^1.128.0",
-    "sinon": "^7.5.0",
+    "simple-git": "^1.129.0",
+    "sinon": "7.3.2",
```

Comment:
did you need to downgrade sinon version?

Reply:
Yes. We don't use sinon, it's just a transitive dependency and it was complaining that the version was out of range. Nothing big, but the warning bothered me.